writeKML.js

```diff
@@ -54,7 +54,20 @@ module.exports = {
         const kmlpaths = [];
         const links = {};
 
+        // Loop through nodes list, generate KML Placemark for each
         update.nodes.forEach(node => {
+
+            // Get band string from first digit of frequency field
+            var freq1 = node.meshrf && String(node.meshrf.freq)[0] || 'None';
```

Comment:
freq1 is constant, so use 'const' here rather than 'var'.
Also, 'var' is never used in JS these days. 'let' is the new 'var'.

Reply:
Got it, updating. 
Interesting... I was taught that constants were only things that stayed the same through the whole program.  In this case freq1 is potentially different for each run of the loop.  But if that's the current practice, happy to use const.

Reply:
This is constant within the scope the variable is valid.